testsuite/objects/__init__.py

```diff
@@ -1,12 +1,44 @@
 """Module containing base classes for common objects"""
 import abc
-from dataclasses import dataclass
+from dataclasses import dataclass, is_dataclass, fields
+from copy import deepcopy
 from functools import cached_property
-from typing import Literal, List
+from typing import Literal, List, Any
 
 from testsuite.objects.sections import Metadata, Identities, Authorizations, Responses
 
```

Comment:
I would also consider whether is is necessary require docstrings here. It is good idea to have reference to struct from go code in docstring, however I feel quite a few classes and functions/methods won't require any documentation in this module.

maybe couple of extra global pylint disables (invalid-name, missing-docstrings) make sense